hugolib/site.go

```diff
@@ -976,7 +977,8 @@ func (s *Site) initializeSiteInfo() {
 		sectionPagesMenu:               lang.GetString("sectionPagesMenu"),
 		GoogleAnalytics:                lang.GetString("googleAnalytics"),
 		BuildDrafts:                    s.Cfg.GetBool("buildDrafts"),
-		canonifyURLs:                   s.Cfg.GetBool("canonifyURLs"),
+		CanonifyURLs:                   s.Cfg.GetBool("canonifyURLs"),
```

Comment:
Yes, but if I correctly understand you, PathSpec hasn't got any public methods to access these fields. That's why we should have these fields in SiteInfo